docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -34,7 +34,7 @@ Run generation using OpenVINO GenAI
 
       python -m venv run-npu-env
       run-npu-env\Scripts\activate
-      pip install openvino>=2024.3.1 openvino-tokenizers>=2024.3.1 openvino-genai>=2024.3.1
+      pip install --pre openvino==2024.3.0.dev20240807 openvino-tokenizers==2024.3.0.0.dev20240807 openvino-genai==2024.3.0.0.dev20240807 --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release
```

Comment:
it's allowed to provide only release versions

Reply:
The latest available version that contains NPU changes is `2024.3.0.dev20240807`. There is no `2024.3.1` unfortunately